actions/actions_whitebox_test.go

```diff
@@ -0,0 +1,180 @@
+package actions
+
+import (
+	"testing"
+
+	"github.com/fabric8-services/fabric8-wit/gormtestsupport"
+	"github.com/fabric8-services/fabric8-wit/resource"
+	tf "github.com/fabric8-services/fabric8-wit/test/testfixture"
+	"github.com/fabric8-services/fabric8-wit/workitem"
+
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestSuiteAction(t *testing.T) {
+	resource.Require(t, resource.Database)
+	suite.Run(t, &ActionSuite{DBTestSuite: gormtestsupport.NewDBTestSuite()})
+}
+
+type ActionSuite struct {
+	gormtestsupport.DBTestSuite
+}
+
+func createWICopy(wi workitem.WorkItem, state string, boardcolumns []interface{}) workitem.WorkItem {
```

Comment:
You're missing `relationShipsChangedAt` field. Please use the `copier` library.

Reply:
Adding a new external library dependency to replace 12 lines of code used in a test does not make sense.

Reply:
Why doesn't it make sense? There is a possibility that we might miss some attributes with manual copy.

Reply:
> Adding a new external library dependency to replace 12 lines of code used in a test does not make sense.

It makes perfect sense because your code doesn't do what it says. It is not creating a copy but it is also modifying the work item by setting two new values. Let's just not add those boilerplate functions, okay? They've cause a huge amount of problems in the past and people started using such functions in other tests. Can you imagine how hard it is if your function is suddenly burried inside some other test code that relies on a perfect copy of a work item and then a field is not set properly?